source/index.html.md

```diff
@@ -2653,7 +2660,9 @@ Receive funds from a Contact by allowing them to pay to a personalised PayID or
     <li><strong>alias_name</strong>: the business name that will be displayed to your customers upon PayID resolution. We suggest using a shortened name appropriate for mobile displays</li>
 </aside>
 <aside class="notice">
-  When creating this type of Contact, the initial response <code>payid_details.state</code> value will always be <code>pending</code>. After a few seconds, it will transition to <code>active</code>. We suggest you use webhooks to be informed of this state change
+  There are two strategies supported for PayID assignment when creating this type of Contact:
+  <li><strong>On-demand PayID activation</strong>: provide <code>payid_email</code> to activate a PayID with the given email address. The initial response <code>payid_details.state</code> value will always be <code>pending</code>. After a few seconds, it will transition to <code>active</code>. We suggest you use webhooks to be informed of this state change.</li>
```

Comment:
```suggestion
  <li><strong>On-demand PayID</strong>: provide a <code>payid_email</code> and we'll create a contact and register a PayID with the given email address. The PayID registration process happens when the request is received. The initial response for <code>payid_details.state</code> will always be <code>pending</code>. It will transition to <code>active</code> when the PayID registration process is complete. This can take up to a few seconds. You can use webhooks to be informed of this state change.</li>
```